mne/viz/topomap.py

```diff
@@ -2738,15 +2739,21 @@ def plot_bridged_electrodes(info, bridged_idx, ed_matrix, title=None,
     -------
     fig : instance of matplotlib.figure.Figure
         The topoplot figure handle.
+
+    Notes
+    -----
+    See also :func:`mne.preprocessing.compute_bridged_electrodes` to determine
+    ``bridged_idx`` and ``ed_matrix``.
```

Comment:
The function name should go into a See Also section

Reply:
For the `See Also` section, it looked to me like usually it only contains function x-links without any text. Can you also put 'comment' like this one which mentions 2 of the arguments in the `See Also` section?